templates/index.tpl

```diff
@@ -25,8 +25,8 @@
                         {t}Audio only (MP3){/t}
                     </label>
                     <div class="seekOptions">
-                        {t}From{/t} <input type="text" placeholder="00:00:00" value="" name="from" />
-                        {t}to{/t} <input type="text" placeholder="00:22:30" value="" name="to" />
+                        {t}From{/t} <input type="text" pattern="(0[0-9]|1[0-9]|2[0-3])(:[0-5][0-9]){2}" placeholder="00:00:00" value="" name="from">
```

Comment:
This pattern only accept hours (00:00 to 24:00). ffmpeg timestamps are structured like this: `HOURS:MM:SS.MILLISECONDS`.
I think we can use `pattern="(\d+:)?(\d+:)?(\d+\.?\d+)"`.

Reply:
This pattern accepts format: HH:MM:SS

Reply:
Yes, but only with with HH < 24 and MM < 60 (which is OK for a time input but not for a duration like we need here).

ffmpeg accepts a wide range of formats. These should all be valid:
* `70` (70 seconds)
* `2:20` (2 minutes and 20 seconds)
* `35:10:20` (35 hours, 10 minutes and 20 seconds)
* `10:20.35` (10 minutes, 20 seconds and 35 milliseconds)

Which is why I suggested a broader pattern.

Reply:
Got it 

But this cause more confusing to users.

What you think?

Reply:
I don't think that is a problem with the new placeholder. Most users will use the format from the placeholder but advanced users can use more complex formats.